packages/docs/docs/react/use-subscription.md

```diff
@@ -0,0 +1,235 @@
+---
+sidebar_position: 3
```

Comment:
Not sure what's up with the sidebar order, but this page currently shows up first in the preview — we may want to check the numbering for all pages in this section and make sure they're sensible

Reply:
The numbering is kind of arbitrary indicator of roughly what order the pages should appear in. 3 is technically first right now but adding an actual 1 would demote its position since 1 is obviously lower. 

I haven't seen a clear way for deciding the numbers, to me it's kinda like hinting "this should probably go last even if there are more pages in the future". 🤷 Maybe should do some research on what other projects do